shared/wallets/common/wallet-popup.js

```diff
@@ -16,39 +17,54 @@ type WalletModalProps = {|
 |}
 
 const WalletPopup = (props: WalletModalProps) => (
-  <Kb.Box2
-    direction="vertical"
-    fullHeight={true}
-    fullWidth={true}
-    centerChildren={true}
-    style={Styles.collapseStyles([styles.container, props.containerStyle])}
-  >
-    {props.children}
-    {props.bottomButtons &&
-      props.bottomButtons.length > 0 && (
-        <Kb.Box2 direction="vertical" style={styles.buttonBarContainer} fullWidth={true}>
-          <Kb.ButtonBar
-            direction={Styles.isMobile ? 'column' : 'row'}
-            fullWidth={Styles.isMobile}
-            style={styles.buttonBar}
-          >
-            {props.bottomButtons}
-          </Kb.ButtonBar>
-        </Kb.Box2>
-      )}
+  <Kb.Box2 direction="vertical" style={styles.outerContainer}>
+    {props.onBack && <Kb.HeaderHocHeader onBack={props.onBack} headerStyle={styles.header} />}
```

Comment:
Having an `onBack` prop in `WalletPopup` will be useful in cases such as closing the create new account and link existing account popups when they're opened from the send form.